localisation/00_wcgw_localisation.csv

```diff
@@ -0,0 +1,3 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
+garfang;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
+garfang_desc;Garfang was a cunning and intelligent gnoll alpha of the Redridge pack, who unified the gnoll tribes and began launching bold attacks on the kingdom of Stormwind, beginning the Gnoll War.
```

Comment:
Duplicate.

If it is not a submod, I would just put it into `00_zuwc_united_localisation.csv` instead.

Reply:
done